src/getSymlinkedDependencies.js

```diff
@@ -1,7 +1,8 @@
 const fs = require('fs')
+const _ = require('lodash')
 
 const isSymlink = dependency =>
     fs.lstatSync(`node_modules/${dependency}`).isSymbolicLink()
 
 module.exports = directory =>
-    fs.readdirSync(`${directory}/node_modules`).filter(isSymlink)
+    _.keys(require(`${directory}/package.json`).dependencies).filter(isSymlink)
```

Comment:
could you remove the `lodash` dependency, the core `Object.keys` function should do the exact same thing here. Just add a check, that dependencies is not null: eg ```Object.keys(require(`${directory}/package.json`).dependencies || {})```

Reply:
👍